### ISSUE TITLE
Are you using Kysely in production? Let us know!

### ISSUE THREAD

**Issue by fhur** [koskimas/kysely#320]
# Are you using Kysely in production?

We're want to add a **Kysely in production** section to our website.

If your company is using Kysely or you know of a company using Kysely in production, **please let us know**.

## Why?
- Showing production use cases of Kysely will bring more trust into the project, and over time more usage.
- Bringing more trust is important to convince larger companies to user Kysely.
- It only takes you 1min to copy and paste the URL of your company 😬 

**Support kysely**! All it takes is a copy-paste 🎉

**Comment by rubiagatra** [comment]
Yes! @zero-one-group uses it in multiple projects, we have been using it since version `0.17.1` until the latest version. @fhur @igalklebanov @koskimas

**Comment by dubscode** [comment]
Happy to say we are using it in two different production apps at Best Western Hotels & Resorts [BW Corporate](https://www.bestwestern.com/en_US/about.html) since around December 2021 :-)

Great library, thank you for your efforts on it!

**Comment by remorses** [comment]
Using it for https://notaku.so, already powering hundreds of websites for our users, awesome developer experience!

**Comment by jtmarmon** [comment]
We're happy users at https://resync.ai !

**Comment by seveibar** [comment]
Hi, we're happily using Kysely in production at https://seam.co

**Comment by joseferben** [comment]
Hello, I am using Kysely for https://www.performance90.com/
It's a joy to use!

**Comment by slvssb** [comment]
Hi, we're using it in https://popstream.co

**Comment by daffron** [comment]
Yeap - nearly finished converting over:

https://paidnice.com

**Comment by heysanil** [comment]
we're launching Kysely in production powering our entire backend as well @ [slingshot.fm](https://slingshot.fm)

thanks for everyone putting in work here, truly a fantastic library (especially in edge environments)

**Comment by ethanresnick** [comment]
We use kysely at getcove.com, and it's been wonderful!

**Comment by tekpriest** [comment]
we use it at https://onah.africa.
Great library 🎸

**Comment by franciscohanna92** [comment]
We would at getlisto.io and kontactless.io, if it only supported SQL Server 😅

**Comment by maktouch** [comment]
We use it https://founderpath.com, and we also forked kysely-codegen to output zod types! 

https://github.com/maktouch/kysely-zod-codegen

**Comment by RobinClowers** [comment]
The admin console (only visible to customers) of https://materialize.com/ uses Kysely, loving it so far!

**Comment by jacobwgillespie** [comment]
We use Kysely at [depot.dev](https://depot.dev)! 

Our DB is PlanetScale today, which we connect to via `mysql2` in our Node services and via `@planetscale/database` in some Cloudflare Workers, and for those we wrote the https://github.com/depot/kysely-planetscale dialect 🚀

**Comment by chukwumaokere** [comment]
We use kysely on https://pixelbypixel.studio

**Comment by BaptisteMahe** [comment]
Hello ! 
We are using Kysely in production at [unicofrance](https://www.unicofrance.com/) !

**Comment by igalklebanov** [comment]
Worth mentioning I "dog fooded" Kysely @ [finaloop](https://www.finaloop.com/) for at least 6 months in production before I left. Hope to one day get buy-in to use it again.

**Comment by thekannman** [comment]
We're using it at [PeakSerp](https://peakserp.com/) and it's been an amazing experience!

**Comment by djMax** [comment]
We are happily using Kysely at [sesamecare.com](https://sesamecare.com)!

**Comment by apuchitnis** [comment]
We're using it for [https://shelf.im/](https://shelf.im/)!

**Comment by alexander-densley** [comment]
using in production for subtract.pro - new landing page drops this week

**Comment by akiver** [comment]
I'm using Kysely for [CS Demo Manager](https://github.com/akiver/cs-demo-manager), an open-source cross-platform desktop app that relies on a PostgreSQL database.
Thank you https://github.com/koskimas, https://github.com/igalklebanov and crontributors ❤️

**Comment by BoscoDomingo** [comment]
Here at [Samara Energy](https://samara.energy) we're also using Supabase in prod!

**Comment by vicary** [comment]
We are using Kysely in production at [OmniWe](https://omniwe.com)!

**Comment by niels-busch-maersk** [comment]
We are using Kysely in production at [Maersk](https://www.maersk.com) inventory control!

**Comment by Rohithzr** [comment]
We hav been using Kysely in production at [PropulsionAI](https://propulsionhq.com) since Day 0. We also use [prisma](https://www.prisma.io) for schema management with [prisma-kysely](https://github.com/valtyr/prisma-kysely) for automatic type generation.

**Comment by nezudevv** [comment]
I am about to use it in production! The only issue I am running into is that my ci/cd fails due to the generate types from kysely-codegen not existing at the time the pipeline runs, and we cannot connect this pipeline to a db for separate reasons. Has anyone run into this as well? If so, what was your solution? Admittedly, I am a bit naive in the area of ci/cd.

**Comment by RobinClowers** [comment]
@nezudevv You should check your generated types in and skip generating them in CI.

**Comment by RyanMabrouk** [comment]
We have been using Kysely at [[SoftyEducation](https://app.softyeducation.com/)](https://app.softyeducation.com/) for the past three months, and it has been performing exceptionally well!

**Comment by alsiola** [comment]
In production at [Percy Main Cricket and Sports Club](https://www.percymain.org), using LibSqlDialect and Turso.

**Comment by rhyzzor** [comment]
We're using Kysely at [GBM Tech & Control](https://www.gbmtech.com.br/en-us) and it was a unanimous decision to refactor all our Raw Queries to Kysely. I just love this tool and it's my choice for 100% of projects

**Comment by byzyk** [comment]
Using Kysely at [Fomr](https://fomr.io) on both server-side and client-side (sync engine) and couldn't be happier!

**Comment by JeongJuhyeon** [comment]
Using Kysely in production for [TOPIK Easy6](https://topikeasy6.com/) as well as [Langda](https://link.oibori.com/ApLM), it's simply best-in-class among Typescript query builders.

**Comment by sean-brydon** [comment]
We are using Kysely for some of our more complex queries in [Cal.com](www.cal.com) where we needed more control of the sql generated.

**Comment by johanneskares** [comment]
[talentir.com](talentir.com) all running on kysely

**Comment by dwoodwardgb** [comment]
Using it to cache data shown on an internal dashboard at Disney/Hulu!

**Comment by iam-abdul** [comment]
using it in prod since day zero at www.spacesos.com

**Comment by oorestisime** [comment]
https://bold.org/ is using it in production. We are slowly migrating from Knex to Kysely

**Comment by AliBakerSartawi** [comment]
At www.open.cx, We migrated fully from TypeORM to Kysely over a year ago and never looked back... Kysely is easily one of the top DX tools in the ecosystem

**Comment by jonnyleeharris** [comment]
We use Kysely in production for our bespoke ERP systems. We are slowly replacing Knex and Prisma.

**Comment by IndigoSoftwares21** [comment]
Yes, we use Kysely on https://www.sabivendor.com empowering 5k+ Small Businesses. It's been great so far.

**Comment by webstyle** [comment]
We use Kysely on https://us.uz for our all Docflow Workflow projects.

**Comment by kerwan-josepavel** [comment]
We use Kysely in production at [Cyna](https://cyna.security/). 🤝
